src/docfx/build/Build.cs

```diff
@@ -57,7 +59,10 @@ private static bool BuildDocset(string docsetPath, string? outputPath, CommandLi
 
                 new OpsPreProcessor(config, errorLog, buildOptions).Run();
                 var sourceMap = new SourceMap(new PathString(buildOptions.DocsetPath), config, fileResolver);
-                errorLog.Configure(config, buildOptions.OutputPath, sourceMap);
+                var validationRules = JsonUtility.DeserializeData<Dictionary<string, ValidationRules>>(
```

Comment:
move this line into `ErrorLog.Configure` method because `ErrorLog.Configure` is also used in `Restore.cs`

Reply:
As we disscussed offline, now restore read the validationRules may cause some problem, so we only read it in Build.cs